<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>dspInventoryAvailability</class>
 <widget class="QWidget" name="dspInventoryAvailability">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventory Availability</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="_asofLit">
     <property name="text">
      <string>As of:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="XComboBox" name="_asof">
     <item>
      <property name="text">
       <string>Item Site Lead Time</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Look Ahead Days</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cutoff Date</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dates</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QStackedWidget" name="_stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="_leadtimePage"/>
     <widget class="QWidget" name="_lookaheadPage">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSpinBox" name="_days">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>290</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_cutoffPage">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="DLineEdit" name="_date"/>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_datesPage">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="DLineEdit" name="_startDate"/>
       </item>
       <item>
        <widget class="QLabel" name="_toLit">
         <property name="text">
          <string>to</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DLineEdit" name="_endDate"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>15</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="_showGroup">
     <property name="title">
      <string>Show</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="XCheckBox" name="_showShortages">
          <property name="text">
           <string>Shortages</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="XCheckBox" name="_showReorder">
          <property name="text">
           <string>Reorder Exceptions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="XCheckBox" name="_ignoreReorderAtZero">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Ignore Reorder at 0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="XCheckBox" name="_byVendor">
          <property name="text">
           <string>By Source Vendor</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>_asof</zorder>
  <zorder>_asofLit</zorder>
  <zorder>horizontalSpacer_2</zorder>
  <zorder>_stackedWidget</zorder>
  <zorder>_stackedWidget</zorder>
  <zorder>_asofLit</zorder>
  <zorder>_asof</zorder>
  <zorder>_stackedWidget</zorder>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>DLineEdit</class>
   <extends>QWidget</extends>
   <header>datecluster.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XCheckBox</class>
   <extends>QCheckBox</extends>
   <header>xcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>XComboBox</class>
   <extends>QComboBox</extends>
   <header>xcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_days</tabstop>
  <tabstop>_date</tabstop>
  <tabstop>_showReorder</tabstop>
  <tabstop>_showShortages</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_asof</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>_stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
